qrisk/stats.py

```diff
@@ -499,57 +569,72 @@ def alpha_beta(returns, factor_returns):
     float
         Beta.
 
-"""
+    """
+    if len(returns) < 2:
+        return (np.nan, np.nan)
```

Comment:
parens are not needed here (only commented since it is inconsistent with the return below)